accounts/views.py

```
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Ruff (0.11.9)

`django.shortcuts.render` imported but unused

Remove unused import: `django.shortcuts.render`

(F401)